scalyr_agent/copying_manager/worker.py

```diff
@@ -1156,6 +1156,10 @@ def FixedRebuildProxy(func, token, serializer, kwds):
         six.ensure_str("add_missing_attributes"),
         six.ensure_str("get_log_path"),
         six.ensure_str("generate_status"),
+        six.ensure_str("add_sampler"),
+        six.ensure_str("add_redacter"),
+        six.ensure_str("skip_to_end"),
+        six.ensure_str("set_new_scalyr_client"),
```

Comment:
Is this one really invoked across the mutliprocess boundary?  I thought the workers create their own scalyr client instance.